### ISSUE TITLE
Working armor for Players

### ISSUE THREAD

**Issue by centry-no** [studio-minus/madness-interactive-reloaded#115]
**Description**
If Arena Mode ever comes out, I hope it will be similar to the MPN:C arena. So, if this is true it would be great if you could buy actual working armor for your character.
If Arena Mode is FALSE, it would also be great if you could just toggle on "Armor" in your Character Customization, so every armored accessory will have the ricochet thing.

**Reasoning**
Because its NOT fair! NPC's engineer mask can make a 5.45 bullet ricochet off their armor and players CAN'T.

**Additional context**
Basically, when you run out of dodge bar there is a chance that their bullet will ricochet off your armor and you will a little of the dodge bar back.
